en/docs/monitoring/api-analytics/moesif-analytics/moesif-integration-guide.md

```diff
@@ -0,0 +1,183 @@
+# Moesif Analytics Integration Guide
+
+This guide outlines the steps required to integrate **Moesif Analytics** with **WSO2 API Manager**. The integration enables you to collect and publish API analytics data to the **Moesif dashboard**, providing insights into API usage, traffic trends, and error tracking.
+
+!!! Note
+    Moesif Analytics integration is supported starting from WSO2 API Manager 4.5.0 Update Level 11 and onwards. Ensure that you are using a compatible update level of WSO2 API Manager 4.5.0.
+    Refer to this [guide](https://apim.docs.wso2.com/en/latest/administer/updating-wso2-api-manager/) to update your WSO2 API Manager to the required or latest update level.
+
+<a href="{{base_path}}/assets/img/analytics/moesif/moesif-data-flow.png"><img src="{{base_path}}/assets/img/analytics/moesif/moesif-data-flow.png" alt="Moesif integration diagram"></a>
+
+### Analytics Data Flow
+
+The Moesif integration captures both **successful** and **failed** API invocations in WSO2 API Manager and publishes them to your Moesif dashboard in near real-time. This allows for powerful analytics and monitoring capabilities.
+
+This section will cover necessary steps to integrate Moesif analytics with WSO2 API Manager.
+
+### Step 1: Set Up Your Moesif Account
+
+#### 1.1 Create an Account and Log In
+
+1. Go to [Moesif’s official website](https://www.moesif.com/)  
+2. Sign up for a new account or log in to your existing account.
+
+#### 1.2 Create a New Application
+
+1. Once logged in, create a **new App** from the “Get Started” page.  
+2. Follow the setup steps and click **Next**.  
+3. You’ll be presented with an **Application ID**. Copy this ID as it will be used in the WSO2 configuration.
+
+!!! note
+    For more detailed instructions and advanced configuration options, refer to the official [Moesif Documentation](https://www.moesif.com/docs).
+
+### Step 2 - Configure WSO2 API Manager
+
+To enable Moesif analytics in WSO2 APIM, you need to update the `deployment.toml` configuration file.
+
+#### Step 2.1 - Configuring the deployment.toml file.
+
+The Choreo based analytics will be enabled by default. Specify the `type` as `moesif` to enable Moesif analytics as shown below. Open the `<APIM-HOME>/repository/conf` directory. Edit the `apim.analyics` configuration in the `deployment.toml` file with the following configuration.
```

Comment:
Open the `<APIM-HOME>/repository/conf` directory. Edit the `apim.analyics` configuration in the `deployment.toml` file with the following configuration. --> Open the `<APIM-HOME>/repository/conf/deployment.toml` file and edit the `apim.analyics` configuration as shown below.